### ISSUE TITLE
unable to connect cloud instance

### ISSUE THREAD

**Issue by ReynoldsProductions** [josephdadams/TallyArbiter#84]
So I have a raspberry pi running on my network and another server running up on amazon in an EC2 free tier instance. They were working fine for a week or so but now the local server is throwing an error saying it is unable to connect to the public EC2 server with the following error 

2021-05-12T01:22:42.669Z] Cloud Connection Disconnected: ___IP and port_____
[2021-05-12T01:22:44.338Z] An error occurred with the connection to ___IP and port_____ Error: xhr poll error
[2021-05-12T01:22:44.350Z] Cloud Connection Disconnected: ___IP and port_____
[2021-05-12T01:22:45.697Z] An error occurred with the connection to ___IP and port_____ Error: xhr poll error
[2021-05-12T01:22:45.709Z] Cloud Connection Disconnected: ___IP and port_____

I have a fairly standard config, 1 atem local to the server and then the cloud instance to serve up to talent remotely.

**Comment by josephdadams** [comment]
Can you still access the TA cloud instance GUI?

**Comment by JTF4** [comment]
If you can still access the GUI, than another good thing to check would be your security policy for the instance. I've had a few cases where my security policy would randomly block ports that I had opened earlier. 

If you're using the dns name that's included with the server, then I would recommend that your assign an elastic IP. I found that it fixed a lot of my issues.

**Comment by ReynoldsProductions** [comment]
yes I can still access the web interface for the cloud instance. 

Not sure where I would make DNS adjustments to the server. I can check about the elastic IP, I don't believe that is enabled on EC2

**Comment by josephdadams** [comment]
Are the socket io versions the same?

Are you running load balancing on the ec2?
